### PR TITLE
make navbar responsive with all devices

### PR DESCRIPTION
**Pull request by AmrNabel**

### CODE REVIEW

File: src/components/Navbar/Responsive.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  const [searchVal, setSearchVal] = useState<string>(query || '')
+  const openMenu = Boolean(anchorEl)
+
+  const handleNavigate = (path: string) => {
+    setOpen(false)
+    setAnchorEl(null)
+    navigate(path)
+  }
+
+  const handleSearch = async () => {
```
Comment (sahdasamier) [inline src/components/Navbar/Responsive.tsx:24]: remove junk code

File: src/containers/Login/index.tsx [hunk 2]
```diff
@@ -134,18 +157,58 @@ const Login = ({ toggleSnackbar, setSnackbarMessage, setLoading }: ILoginProps)
...
+  const windowWidth = window.innerWidth
+  const elementWidth = `calc(${windowWidth}px - 50%)`
+
   return (
     <>
       <img src={polygon1} alt='' style={{ position: 'absolute', top: '3%', left: '-10%' }} />
       <img src={polygon2} alt='' style={{ position: 'absolute', top: '50%', right: '20%' }} />
       <img src={polygon3} alt='' style={{ position: 'absolute', right: '20%', top: '5%', width: '200px' }} />
+      {isSmallScreen ? (
+        <>
```
Comment (sahdasamier) [inline src/containers/Login/index.tsx:170]: {!isSmallScreen && } not using null it is more  better

File: src/containers/Register/index.tsx [hunk 3]
```diff
@@ -39,11 +63,51 @@ const Register = ({ toggleSnackbar, setSnackbarMessage, setLoading }: IRegisterP
...
     }
   })
+  const isSmallScreen = useMediaQuery('(max-width:819px)')
+  const windowWidth = window.innerWidth
+  const elementWidth = `calc(${windowWidth}px - 50%)`
   return (
     <>
       <img src={polygon1} alt='' style={{ position: 'absolute', top: '3%', left: '-10%' }} />
       <img src={polygon2} alt='' style={{ position: 'absolute', top: '50%', right: '20%' }} />
       <img src={polygon3} alt='' style={{ position: 'absolute', right: '20%', top: '5%', width: '200px' }} />
```
Comment (sahdasamier) [inline src/containers/Register/index.tsx:73]: the same at login too , "about null"

Comment (sahdasamier) [inline src/containers/Register/index.tsx:73]: the same at login too , "about null"

File: src/containers/Search/index.tsx [hunk 4]
```diff
@@ -219,7 +225,42 @@ const Search = (homeProps: IHomeProps) => {
         setSnackbarMessage={setSnackbarMessage}
         toggleSnackbar={toggleSnackbar}
       />
-
+      {isSmallScreen ? (
```
Comment (sahdasamier) [inline src/containers/Search/index.tsx:228]: the same of login and register "about null"

**Review by sahdasamier — APPROVED** [review]

File: src/containers/Search/index.tsx [hunk 5]
```diff
@@ -207,6 +210,9 @@ const Search = (homeProps: IHomeProps) => {
       document.removeEventListener('contextmenu', event => event.preventDefault())
     }
   }, [])
+  const windowWidth = window.innerWidth
+  const elementWidth = `calc(${windowWidth}px - 50%)`
```
Comment (ArnobChowdhury) [inline src/containers/Search/index.tsx:214]: Don't calculate width like this. Give the search bar the same size as the form in smaller screen.

**Review by ArnobChowdhury — CHANGES_REQUESTED** [review]
You are repeating the same code in `Login`, `Register`, `Search`. DRY. Make the search bar a different component. And use that component wherever you need to.

File: src/App.css [hunk 6]
```diff
@@ -6,11 +6,15 @@
...
-  justify-content: center;
   font-size: calc(3px + 2vmin);
   color: rgb(37, 3, 3);
 }
 
+@media (min-width: 819px) {
+  .app {
+    justify-content: center;
+  }
+}
```
Comment (ArnobChowdhury) [inline src/App.css:17]: Instead of writing breakpoints like this use the breakpoints given by Material UI.

**Review by sahdasamier — CHANGES_REQUESTED** [review]
hi Amr, there are some files you deleted as Jason oR composed and another, so please recreate this PR , because there are more conflicts here because of that 
so I will close and sorry for that

**Pull request closed without merging**